JitsiMeetJS.ts

```diff
@@ -495,6 +509,104 @@ export default {
         return Statistics.audioLevelsEnabled && LocalStatsCollector.isLocalStatsSupported();
     },
 
+    /**
+     * Simple way to create a {JitsiConference}. All options are optional and sane defaults
+     * will be chosen for JaaS users.
+     *
+     * @param roomName - The name of the conference.
+     * @param appId - The application id (also known as tenant).
+     * @param token - The token (JWT) to use for authentication.
+     * @param options - The options to use for joining the conference.
+     */
+    async joinConference(
+            roomName: string,
+            appId: string = '',
+            token: string | null = null,
+            options: IJoinConferenceOptions = {}): Promise<JitsiConference> {
+        const d = new Deferred();
+        let connectionOptions = options.connectionOptions ?? {};
+
+        // Provide a solid default config in case of JaaS.
+        if (appId.startsWith('vpaas-magic-cookie')) {
+            // Auto-initialize the library.
+            const autoInit = Symbol.for('kLJMAutoInit');
+
+            if (typeof window[autoInit] === 'undefined') {
+                window[autoInit] = true;
+
+                // TODO(saghul): review rtcstats initialization.
```

Comment:
@andrei-gavrilescu I really don't like how this turned out, unless I add this here rtcstats won't be initialized, but I think it should, if the connection initialization is what happens first.

Reply:
If I rmmbr correctly the reason why we need to initialize rtcstats before LJM gets loaded, is that rtcstats proxies calls to the WebRTC/GUM api, something like  `window[${prefix}RTCPeerConnection] = peerconnectionproxy`, and LJM does something similar where it uses a reference to `window[${prefix}RTCPeerConnection]`, 
If rtcstats doesn't get initialized before that, then LJM gets the reference to the non proxied RTCPeerConnection ctor and uses it in subsequent calls.
That's the reason we need to pass it on init, and the reason we need to pass the config to every connection as well is that it can change in between connections where LJM doesn't get reloaded, such as electron.

Reply:
We could have rtcstats always proxy the WertcAPI, and simply have it not connect to the server.
So we would only need to pass the config once, the implication would be that in cases where rtcstats is disabled, the proxies would still be active but not sending anything.

Reply:
looking through LJM a bit, that might no longer be the case, seems like we use RTCPeerConneciton directly now, so removing it might work, need to do some checks.

Reply:
Thank you!